sdk/storage/src/core/mod.rs

```diff
@@ -40,8 +40,8 @@ pub struct IPRange {
 
 pub use stored_access_policy::{StoredAccessPolicy, StoredAccessPolicyList};
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
```

Comment:
I'm not sure serde's default is what you are expecting. Can you please add a test for test for serialize & deserialize of `ConsistencyCRC64` & `ConsistencyMD5`?

Reply:
Good catch, I fixed the XML deserialization (used in the List Blobs API).

And I removed the `Serialize` derive for it seems not used.